drivers/clock_control/clock_stm32_ll_u5.c

```diff
@@ -79,6 +79,13 @@ static uint32_t get_startup_frequency(void)
 		return get_msis_frequency();
 	case LL_RCC_SYS_CLKSOURCE_STATUS_HSI:
 		return STM32_HSI_FREQ;
+	case LL_RCC_SYS_CLKSOURCE_STATUS_HSE:
+		/* HSE is un-expected freq */
+		return STM32_HSE_FREQ;
```

Comment:
Question: if HSE is unexpected, shouldn't we also have an __ASSERT()?

Reply:
indeed, _ASSERT_ is the cause of crash "ASSERTION FAIL"